General-Features.md

```diff
@@ -19,7 +19,24 @@ It's possible to hide the sidebar by pressing the left arrow next to the minimiz
 
 ### Profiles
 
-As of the 1.9.11 release, profiles are now available to hold different distinct settings configurations within the same client instance.  By default the profile loaded on startup will be the one that was most recently active, to override this you can pass `--profile=<profilename>` to the launcher and it will load that specific profile on startup.
+Profiles are separate sets of plugins and settings that can be switched between at any time. They can be managed under a tab in the configuration panel.
+
+![](img/profiles.png)
+
+Below each profile are buttons to interact with them. Toggled buttons will turn orange to indicate they are enabled. From left to right, the buttons are: 
```

Comment:
Technically there's also the button that activates/switches to a profile (``>``). Sometimes people have difficulty switching profiles, because they don't understand that they have to click the ``>`` twice if they haven't 'expanded' that profile yet (once to 'expand' it, the second time to actually switch profiles).
Maybe you could add the switch profile button somewhere, and instructions on how to switch profile? That could also include some brief info on how the user knows the profile is active (orange bar).

Rest of the changes LGTM!

Reply:
Added an explanation for the button that expands/activates a profile, and a mention that the currently selected profile is indicated by the orange bar. Thanks for the feedback!